### PR TITLE
Make it clear when you should use this library

### PR DESCRIPTION
**Pull request by sethladd**
from a codelab

### CODE REVIEW

**Comment by sethladd** [comment]
cc @kevmoo @Janamou

**Comment by sethladd** [comment]
thanks @kevmoo ! ptal

File: README.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
-# Dart wrapper library for the new Firebase
+# Firebase REST APIs for Dart
```
Comment (kevmoo) [inline README.md]: Remove `REST APIs` here. Maybe just say Dart server, fuchsia, and browser

**Comment by sethladd** [comment]
Thanks! Updated. ptal

**Comment by kevmoo** [comment]
Thanks, @sethladd !

**Pull request merged**